### ISSUE TITLE
EXCITEBIKE64 stops in pure interpreter

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#280]
Originally reported on Google Code with ID 281

```
Describe your system:
 - Linux distribution: Debian sid
 - Machine type: 64-bit
 - Mupen64Plus version: 1416
 - Plugins used: glide64, jttl_audio, 

Stops when starting a game with the output "NI:7000d" using pure
interpreter and "NI() @ 800128cc" when using interpreter.
This would be
000000  00000001110000000000 001101
Special-Break with Code 00000001110000000000

So. Exception-Break handling must be implemented.
```

Reported by `sven@narfation.org` on 2009-10-04 00:19:58

**Comment by richard42** [comment]
Reported by `richard42g` on 2010-01-13 13:19:23
- **Labels added**: Type-Defect, Component-Core

**Comment by richard42** [comment]
```
It also crashes in Cached Interpreter and dynamic recompiler and with
Core Error: DIV: divide by 0
Core Error: NI() @ 0x800127ec
Core Error: opcode not implemented: 800127ec:7000d
in the title screen.

(Mupen 64plus 1.99.2, OpenGL Video Plugin by Rice' v1.99.2, Ubuntu 9.10 64bit)
```

Reported by `benpicco` on 2010-02-01 12:08:40

**Comment by richard42** [comment]
tested, works for me
